http/GPON_routers.lua

```diff
@@ -0,0 +1,76 @@
+-- lua script for GPON exploit
+
+function canHandle()
+    return true
+end
+
+function handle(message)
+    local url = retrieveURLs(message);
+    if (url and string.match(url, 'gpon.php')) then
+        getFileDownload(url, "downloads")
+        return "wget: missing URL \nUsage: wget [OPTION]... [URL]... \n\nTry `wget --help' for more options.\n";
+    end
+    return message;
+end
+
+function preHandle(message)
+
+end
+
+function afterHandle(message)
+
+end
+
+-- https://stackoverflow.com/questions/23590304/finding-a-url-in-a-string-lua-pattern
+
+-- all characters allowed to be inside URL according to RFC 3986 but without
+-- comma, semicolon, apostrophe, equal, brackets and parentheses
+-- (as they are used frequently as URL separators)
+function retrieveURLs(message)
```

Comment:
This should be a global method in my opinion, since it can be useful to many lua scripts.